manifests/init.pp

```diff
@@ -10,3 +10,14 @@
 # it and/or modify it under the terms of the GNU
 # General Public License version 3 as published by
 # the Free Software Foundation.
+
+class user (
+  $managed_hash = {},
+  ) {
+
+  if $managed_hash != {} {
```

Comment:
i thought that was ${resourcetype_name}_hash =)
i'll change to $managed_users_hash